server/pulp/server/webservices/views/repositories.py

```diff
@@ -182,6 +181,10 @@ def get(self, request, repo_id):
         if request.GET.get('distributors', 'false').lower() == 'true' or details:
             _merge_related_objects(
                 'distributors', manager_factory.repo_distributor_manager(), (repo,))
+        if details:
+            repo['total_repository_units'] = sum(repo['content_unit_counts'].itervalues())
+            total_missing = repo_controller.missing_unit_count(repo_obj)
+            repo['total_downloaded_units'] = repo['total_repository_units'] - total_missing
```

Comment:
Maybe this should have a different name, since some may be uploaded instead of downloaded. Perhaps ``locally_stored_units`` or something similar?